sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -548,52 +573,8 @@ public Mono<Response<BinaryData>> getImageOperationStatusWithResponse(
      */
     @Generated
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<BinaryData, BinaryData> beginStartGenerateImage(
+    PollerFlux<BinaryData, BinaryData> beginBeginAzureBatchImageGeneration(
```

Comment:
We should fix this name in the TypeSpec. "beginBegin"

Reply:
That's odd, the typespec seems to be named correctly: https://github.com/Azure/azure-rest-api-specs/pull/24556/files#diff-6b3ae1ce13863d9ac0f4ccffd8ad252dabab9913f176208796b0252a5fca5723R79

I will surface this in the standup today. Thanks for the catch!